# The optimal motion plan represented in cartesian space

std_msgs/Float64 dt                     # the time [s] between two states
 RobotStateCartesianStamped[] states    # trajectory of states
